checklists.md

```diff
@@ -0,0 +1,293 @@
+<div align="center">
+    <a href="https://github.com/dwyl/product-roadmap">
+        <img src="https://user-images.githubusercontent.com/194400/57011570-34013080-6bfa-11e9-8773-f01bb2c7b932.png">
+    </a>
+</div>
+<br />
+
+# Checklists?
+
+A checklist is the most effective way of
+keeping track of the list of things to be done
+and ensuring that nothing is missed.
+
+https://www.google.com/search?q=checklist
+![image](https://user-images.githubusercontent.com/194400/59164503-85f47a80-8b05-11e9-9799-26446f8aff0c.png)
+> "_a list of items required, things to be done,
+or points to be considered, used as a reminder._"
+
+## The Checklist Manifesto
```

Comment:
Looks like this subtitle is not rendered properly:
![image](https://user-images.githubusercontent.com/6057298/59186068-e0bfbd80-8b69-11e9-80c2-c420892c92a8.png)